LayoutTests/crypto/workers/reportingObserverNonNull.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
```

Comment:
The test in LayoutTests/crypto/workers is not great.
Let's add it to LayoutTests/http/tests/workers